vm/policy.go

```diff
@@ -58,16 +110,11 @@ func PolicyHandling(t string, path string, o PolicyOptions) error {
 		return err
 	}
 
-	client := pb.NewPolicyServiceClient(conn)
-
-	req := pb.Policy{
-		Policy: policyEventData,
-	}
-	if resp, err := client.HostPolicy(context.Background(), &req); err == nil {
-		if resp.Status == 1 {
-			fmt.Print("Success")
-		} else {
-			return fmt.Errorf("failed to send policy")
+	if err = sendPolicyOverHttp(policyEventData); err != nil {
+		// HTTP connection is not active, hence trying to send policy over gRPC
+		if err = sendPolicyOverGrpc(o, policyEventData); err != nil {
+			// Failed to send policy over gRPC
+			return err
```

Comment:
We should add some flag for systemd/control plane mode maybe then default to this

Reply:
Should be fine. In this case, it will try passing the information over http if it is not present then pass over to grpc. So control plane configuration is not required.

Reply:
Need to discuss on the same

Reply:
@nyrahul  Please provide your comments on the same.

Reply:
Yes, we need flag based control ... since failing from HTTP to GRPC can take time i.e. timeouts ... This needs to be handled.